README.md

```diff
@@ -3,3 +3,12 @@
 This project has been created by a student at Project Shift, a software engineering fellowship located in Downtown Durham.  The work in this repository is wholly of the student based on a sample starter project that can be accessed by looking at the repository that this project forks.
 
 If you have any questions about this project or the program in general, visit projectshift.io or email hello@projectshift.io.
+
+
+****************************
+
+The front-end React App assumes a database exists with product data. Start-up procedures are:
```

Comment:
I appreciate these instructions and other users will, too! You forgot to add a .gitignore file to your back-end though, just a heads up!